### DISCUSSION TITLE
Error: "non-hexadecimal number found in fromhex() arg at position 61" when opening my wallet

### DISCUSSION THREAD

**Discussion by blackmennewstyle** [Chia-Network/chia-blockchain#8608] [Support]
OS: GNU/Linux Debian 11
GUI: XCFE4
Wallet: Chia

I started to have that issue after using the wallet for a while, i closed it for few hours and opened it again, i just keep having that error warning:
```
Error: "non-hexadecimal number found in fromhex() arg at position 61" when opening my wallet
```
![Screenshot_2021-09-27_00-03-50](https://user-images.githubusercontent.com/4682874/134828916-d8d3ac92-9e65-4bba-a572-9fe9c7eed9f5.png)

Is my database corrupted or something? How to avoid that situation in the future?

**Comment by loppefaaret** [comment]
perhaps we can get a bit more information from the log found in you users folder `~/.chia/mainnet/log/debug.log` ?

**Comment by blackmennewstyle** [comment]
[debug.log](https://github.com/Chia-Network/chia-blockchain/files/7237353/debug.log)
With great pleasure but i actually deleted all folders and files in the .chia folder earlier on, so the debug.log file might not be helpful anymore :$

**Reply by loppefaaret** [reply]
yeah, if you removed `~\.chia` that includes the log... let me know if it happens again

**Comment by BrandtH22** [comment]
@blackmennewstyle, since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
